src/test/java/qna/domain/AnswerTest.java

```diff
@@ -33,18 +34,22 @@
     void setUp() {
         javajigi = userRepository.save(UserTest.JAVAJIGI);
         sanjigi = userRepository.save(UserTest.SANJIGI);
-        questionJavajigi = questionRepository.save(new Question("title1", "contents1").writeBy(javajigi));
+        questionJavajigi = questionRepository.save(QuestionFixture.질문().writeBy(javajigi));
```

Comment:
```suggestion
        Question question = new Question("title1", "contents1");
        questionJavajigi = questionRepository.save(question.writeBy(javajigi));
```

이런식으로 바꿔보고 테스트 진행해보시고 확인부탁드려요~

영속화 대상의 경우 공통으로 계속 사용하면 안될 것 같습니다.